promtimer/util.py

```diff
@@ -38,7 +38,10 @@ def kill_node(process):
         pass
 
 def start_process(args, log_filename, cwd=None):
-    log_file = open(log_filename, 'a')
+    if log_filename is not None:
+        log_file = open(log_filename, 'a')
+    else:
+        log_file = subprocess.DEVNULL
```

Comment:
what happens if log_file is set to None?

Reply:
> what happens if log_file is set to None?

I first tried that and the output went to stdout. Some web searching found the following...

https://docs.python.org/3/library/subprocess.html

stdin, stdout and stderr specify the executed program’s standard input, standard output and standard error file handles, respectively. Valid values are PIPE, DEVNULL, an existing file descriptor (a positive integer), an existing file object, and None. PIPE indicates that a new pipe to the child should be created. DEVNULL indicates that the special file os.devnull will be used. **With the default settings of None, no redirection will occur; the child’s file handles will be inherited from the parent.**